### ISSUE TITLE
CMake config as subdirectory fails

### ISSUE THREAD

**Issue by ascn** [oneapi-src/oneTBB#548]
When I configure oneTBB as a subdirectory it fails with this error:
```
CMake Error: File my_proj_root/integration/pkg-config/tbb.pc.in does not exist.
CMake Error at build/debug/_deps/onetbb_util-src/src/tbb/CMakeLists.txt:140 (configure_file):
  configure_file Problem configuring file
```
I'm on v2021.3.0.
Changing this line in src/tbb/CMakeLists.txt:140 from
```
    configure_file(${CMAKE_SOURCE_DIR}/integration/pkg-config/tbb.pc.in ${CMAKE_CURRENT_BINARY_DIR}/${TBB_LIB_NAME}.pc @ONLY)
```
to
```
    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/../../integration/pkg-config/tbb.pc.in ${CMAKE_CURRENT_BINARY_DIR}/${TBB_LIB_NAME}.pc @ONLY)
```
fixes this.

I am including oneTBB using FetchContent:
```
FetchContent_Declare(oneTBB_util
        PREFIX            ${EXTERNAL}/oneTBB
        GIT_REPOSITORY    https://github.com/oneapi-src/oneTBB.git
        GIT_TAG           v2021.3.0
        INSTALL_DIR       ${EXTERNAL}/oneTBB/install
        CMAKE_ARGS        -DTBB_TEST:BOOL=OFF -DCMAKE_INSTALL_PREFIX:PATH=${EXTERNAL}/oneTBB/install
        TEST_COMMAND      ""
        LOG_DOWNLOAD      ON
        )
FetchContent_MakeAvailable(oneTBB_util)
```

**Comment by vlserov** [comment]
Issue has been resolved with #476
Please verify in v2021.4.0-rc1 release

**Comment by ascn** [comment]
Works now! Thank you :)
